dev/a11y_assessments/lib/main.dart

```diff
@@ -81,7 +81,13 @@ class HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Accessibility Assessments')),
+      appBar: AppBar(
+        title: Semantics(
+          header: true,
```

Comment:
header and headings are treated differently.

Actually now I think of it, should probably just have header = true without heading level @yjbanov do you think this makes sense? also is header= true supported in web engine?